src/Datadog.Trace/Tagging/TagsList.cs

```diff
@@ -13,7 +13,34 @@ internal abstract class TagsList : ITags
 
         public List<KeyValuePair<string, double>> Metrics => Volatile.Read(ref _metrics);
 
-        public List<KeyValuePair<string, string>> Tags => Volatile.Read(ref _tags);
+        public List<KeyValuePair<string, string>> GetAll()
```

Comment:
Not sure about the name b/c this interface contains not only tags but also metrics. However, I prefer methods in interfaces rather than properties. Maybe `GetAllTags()`?
I think it is better to wait for others' comments before doing any rename.

Reply:
I think I may have suggested `GetAll()` to avoid the stutter, however, what I agree with Robert's suggestion since it is symmetrical with the other `Get*Tags()` methods. Maybe later we look at renaming `ITags` to something else since it also include metrics.